third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -58,6 +58,8 @@ fn test_runner(path: &Path) -> datatest_stable::Result<()> {
     let mut sources = extract_test_directives(path, "// dep:")?;
     sources.push(path.to_string_lossy().to_string());
     let deps = vec![path_from_crate_root("../move-stdlib/sources")];
+    let path_string = path.to_string_lossy();
+    let warn_unused = path_string.contains("unused");
```

Comment:
To mimic the check done in `/third_party/move/move-compiler/tests/move_check_testsuite.rs`, we should change this to `"warn_unused"` instead?

Reply:
Also, should we make it check for `"/warn_unused/"` (i.e., directory check)? Right now, we enable this based one file name as well (eg., `/inlining/unused_inline.move`).

Reply:
Actually, I noticed that
- if we enable warn_unused everywhere then we have a problem
- we already have a few tests with "unused" in their names which might benefit from this check

I guess I hadn't yet finalized whether we need a whole set of tests for this functionality, since (a) someone asked for it, but (b) it's pretty annoying to enable.